src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -576,7 +676,7 @@ private void createLocatableXsvDataSource(final Path dataSourceFile,
         manualFeatureInputs.add(featureInput);
 
         // Create a locatable XSV feature reader to handle XSV Locatable features:
-        final LocatableXsvFuncotationFactory locatableXsvFuncotationFactory = new LocatableXsvFuncotationFactory();
+        final LocatableXsvFuncotationFactory locatableXsvFuncotationFactory = new LocatableXsvFuncotationFactory(name, version, annotationOverridesMap);
```

Comment:
I'm not sure that we want the overrides going into each datasource.  Seems unnecessary...  Wouldn't we want to have the overrides applied after the datasources had generated their funcotations?

Reply:
If you agree, feel free to file an issue instead of addressing immediately.

Reply:
This is the way all the manual annotations have been working since that feature was implemented - I just forgot to hook them into the `LocatableXsvFuncotationFactory` in the construction.

I built them this way because it made overriding fields with VCF much easier.

Because MAF needs a map for the funcotations, it may now be easier to do this in the output renderers themselves.  It will probably simplify a few things.